lisk/Dockerfile

```diff
@@ -3,19 +3,19 @@ FROM node:10-alpine AS builder
 ENV NODE_ENV=production
 
 RUN apk --no-cache upgrade && \
-    apk --no-cache add alpine-sdk python2 libtool autoconf automake
+	apk --no-cache add alpine-sdk python2 libtool autoconf automake
```

Comment:
Why change the indentation? both `apk [...]` commands are not aligned any more now.

Reply:
@fchavant That's prettier's doing actually. It replaced whitespaces with tabs since we are using tabs everywhere. I say let's keep it that way if it doesn't create a technical problem, what do you think?

Reply:
Why does prettier run on files that are not `*.ts`, `*.js` or `*.json`?

Reply:
The prettier extension on my IDE 😅

Reply:
Can it not be configured? I'd really like to keep the `Dockerfile` as-is (more readable; with less changes) if possible.

Reply:
I reverted it.